libsignal-service/src/profile_service.rs

```diff
@@ -22,7 +22,7 @@ impl ProfileService {
         let endpoint = match profile_key {
             Some(key) => {
                 let version = bincode::serialize(
-                    &key.get_profile_key_version(address.aci()),
+                    &key.get_profile_key_version(address.aci().unwrap()),
```

Comment:
```suggestion
                    &key.get_profile_key_version(address.aci().expect("valid profile key version")),
```

I'd try to replace most of the unwraps in this PR with `expect`s